### ISSUE TITLE
[Net\serve] the function returns a SIGSEGV

### ISSUE THREAD

**Issue by RickBarretto** [arturo-lang/arturo#1239]
**Describe the bug**
"seems to break upon requesting a GET to localhost:18966/post?id=123, with SIGSEGV. The first route (parameterless) works fine, but the second and third both crash the arturo process"

**To Reproduce**
```ruby
serve .port:18966 [
GET "/"          [ "This is the homepage" ]
GET "/post"      $[id][ 
                    send.html ~"This is the post with id: |id|" 
                 ]
POST "/getinfo"  $[id][ 
                   send.json write.json ø #[
                     i: id
                     msg: "This is some info"
                   ]
                 ]
]
```

**Expected behavior**
It should work as said [in documentation][docs].

**Desktop (please complete the following information):**
 - OS: Windows, Linux (tested in both)
 - Version: `v0.9.83`

**Additional context**
[This message from Discord][discord], thanks `@Rosen`.

[docs]: https://arturo-lang.io/documentation/library/net/serve/
[discord]: https://discord.com/channels/765519132186640445/829324913097048065/1130602482331025509

**Comment by RickBarretto** [comment]
@drkameleon Wasn't it fixed before at #1240?

**Comment by drkameleon** [comment]
> @drkameleon Wasn't it fixed before at #1240?

You must be right! I'll have a look...

**Comment by drkameleon** [comment]
I've been testing this one and it appears all pending issues have indeed been fixed. So, I'm closing this...

In case the issue persists, we can ofc open it again.
